store/src/java/com/zimbra/cs/mailbox/Mailbox.java

```diff
@@ -10828,4 +10867,47 @@ public void noOp() throws ServiceException {
     public void resetDefaultCalendarId() throws ServiceException {
         getAccount().setPrefDefaultCalendarId(ID_FOLDER_CALENDAR);
     }
+
+    public static Element sendRequest(Account acct, Element request, String uri, boolean authRequired, int timeout)
+            throws ServiceException {
+        assert request != null;
+        assert acct != null;
+
+        String soapUri = null;
+        if (uri == null || uri.length() == 0) {
```

Comment:
use StringUtil.isNullOrEmpty for uri check

Reply:
modified to  StringUtil.isNullOrEmpty check